pkg/lang/csharp/utils_test.go

```diff
@@ -0,0 +1,111 @@
+package csharp
+
+import (
+	"github.com/klothoplatform/klotho/pkg/core"
+	"github.com/stretchr/testify/assert"
+	"strings"
+	"testing"
+)
+
+func Test_IsValidTypeName(t *testing.T) {
+	tests := []struct {
+		name           string
+		typeQuery      string
```

Comment:
This looks the same in all test cases.

Reply:
That's mostly happenstance. There are other node locations where this function can be used. They're just not used in the current test suite.

Reply:
My latest push includes a couple of additional cases using different `typeQuery` values